src/main/java/peoplesoft/model/money/Rate.java

```diff
@@ -37,8 +37,9 @@
 @JsonSerialize(using = Rate.RateSerializer.class)
 @JsonDeserialize(using = Rate.RateDeserializer.class)
 public class Rate {
-    public static final String MESSAGE_CONSTRAINTS = "Value for rate should be a non-negative decimal number.";
-    public static final String MESSAGE_TOO_LARGE = "Value for rate is too large.";
+    public static final String MESSAGE_CONSTRAINTS = "The rate should not be negative. "
+            + "It can include 2 decimal places.";
+    public static final String MESSAGE_TOO_LARGE = "PeopleSoft cannot store such a high rate.";
```

Comment:
hmm i'm not sure about this change, because it implies that our app has an inherent limitation instead of just telling the user that they're potentially doing something Not Good^tm

Reply:
😉